redaxo/src/addons/structure/plugins/content/lib/article_content_editor.php

```diff
@@ -48,7 +48,7 @@ protected function outputSlice(rex_sql $artDataSql, $moduleIdToAdd)
             if ('add' != $this->function && $this->slice_id == $sliceId) {
                 $msg = '';
                 if ('' != $this->warning) {
-                    $msg .= rex_view::warning($this->warning);
+                    $msg .= rex_view::error($this->warning);
```

Comment:
hmm sollen wir wirklich eine message die wir semantisch als "warning" führen, am ende als error ausgeben?

was ist da der hintergrund?

Reply:
> was ist da der hintergrund?

Schau dir das Vorherbild einmal an. Da wird das Alert einmal als Error oberhalb der Slices  und einmal als Warning innerhalb des Moduls ausgegeben. Ich wollte es jetzt vereinheitlichen.

Reply:
In R4 hatten wir nur rex_info und rex_warning. Denke, dass deshalb die Property warning heißt. Erst seit R5 unterscheiden wir zwischen warning und error.